analyses/management/commands/merge_mgys_duplicates.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from analyses.models import Study, Assembly, Run, Analysis
+
+
+class Command(BaseCommand):
+    help = "Identify duplicate MGnify studies with the same ENA study and reassign runs and assemblies."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dry-run",
+            action="store_true",
+        )
+
+    def handle(self, *args, **options):
+        self.dry_run = options["dry_run"]
+        self.print_duplicate_mgys_studies()
+
+    #   find ENA studies with two MGYS accessions
+    def print_duplicate_mgys_studies(self):
```

Comment:
Just slightly better naming since `print_...` suggests this method would be idempotent / not have side effects.
```suggestion
    def handle(self, *args, **options):
        self.dry_run = options["dry_run"]
        self.deduplicate_mgys_studies()

    #   find ENA studies with two MGYS accessions
    def deduplicate_mgys_studies(self):
```